### PR TITLE
VCSWP-19919(2023 Q3 C# Quickstart Updates)

### PR DESCRIPTION
**Pull request by edgeston**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,18 @@ To get started using a FreeClimb account, follow the instructions [here](https:/
    $  dotnet run
    ```
 
+- If you're having issues running the application, remove any other instances of the enviornment varibles being set and paste the ACCOUNT_ID and API_KEY values directly in those fields as strings.
```
Comment (katebrune) [inline README.md]: is this issue only happening with this quickstart?

Comment (edgeston) [reply README.md]: Yes, the other two are working

Comment (katebrune) [reply README.md]: is there anything different about this setup than the other two?

Comment (edgeston) [reply README.md]: Hey @katebrune. Removed exception in the readme and added a launchSettings.json file where env variables can be added. Everything's working as it should now.

**Review by katebrune — APPROVED** [review]

**Pull request merged**